src/Setup/WixBA/RootView.xaml.cs

```diff
@@ -26,6 +26,7 @@ public RootView(RootViewModel viewModel)
             this.InitializeComponent();
 
             viewModel.ViewWindowHandle = new WindowInteropHelper(this).EnsureHandle();
+            WindowProperties.Instance.CheckBackgroundBrightness();
```

Comment:
We should be able to use this DependencyProperty declaratively.  Typically it would be a Boolean property, we set it to true in the xaml, and the property detects when it changes.  In the change handler, it would do `CheckBackgroundBrightness`.